Mlem/Models/Trackers/Post Tracker.swift

```diff
@@ -67,28 +68,50 @@ class PostTracker: FeedTracker<APIPostView> {
 
     private func preloadImages(_ newPosts: [APIPostView]) {
         URLSession.shared.configuration.urlCache = AppConstants.urlCache
-        for post in newPosts {
-            if let thumbnailUrl = post.post.thumbnailUrl {
-                Task(priority: .background) {
-                    await preloadSingleImage(url: thumbnailUrl)
+        for postView in newPosts {
+            // preload user and community avatars--fetching both because we don't know which we'll need, but these are super tiny
+            // so it's probably not an API crime, right?
+            Task(priority: .userInitiated) {
```

Comment:
Why user initiated? I think this should be background

Reply:
Muscle memory lol